### PR TITLE
Rename unit tests to follow convention

### PR DESCRIPTION
**Pull request by unpluggedk**
test<methodName>_when<Condition>_should<Result>

### CODE REVIEW

File: MSAL/test/unit/MSALStringHelperMethodTests.m [hunk 1]
```diff
@@ -48,33 +48,33 @@ - (void)tearDown
...
 }
 
-- (void)testIsStringNilOrBlankNil
+- (void)testMsalIsStringNilOrBlank_whenNil_shouldReturnTrue
 {
     XCTAssertTrue([NSString msalIsStringNilOrBlank:nil], "Should return true for nil.");
 }
 
-- (void)testIsStringNilOrBlankSpace
+- (void)testMsalIsStringNilOrBlank__whenBlankSpaceOnly_shouldReturnTrue
```
Comment (oldalton) [inline MSAL/test/unit/MSALStringHelperMethodTests.m]: nit: double underscore

File: MSAL/test/unit/MSALStringHelperMethodTests.m [hunk 2]
```diff
@@ -48,33 +48,33 @@ - (void)tearDown
...
 }
 
-- (void)testIsStringNilOrBlankMixed
+- (void)testMsalIsStringNilOrBlank_whenMixedWhiteSpaceOnly_shouldReturnTrue
 {
     XCTAssertTrue([NSString msalIsStringNilOrBlank:@" \r\n\t  \t\r\n"], "Should return true for nil.");
 }
 
-- (void)testIsStringNilOrBlankNonEmpty
+- (void)testMsalIsStringNilOrBlank_whenNotEmpty_shouldReturnFalce
```
Comment (oldalton) [inline MSAL/test/unit/MSALStringHelperMethodTests.m]: nit: typo "Falce"

File: MSAL/test/unit/MSALStringHelperMethodTests.m (hunk 1 above)
Comment (unpluggedk) [inline MSAL/test/unit/MSALStringHelperMethodTests.m]: fixed

File: MSAL/test/unit/MSALStringHelperMethodTests.m (hunk 2 above)
Comment (unpluggedk) [inline MSAL/test/unit/MSALStringHelperMethodTests.m]: also fixed, thanks

**Pull request merged**